src/lib/tecs/TECS.cpp

```diff
@@ -731,10 +740,17 @@ void TECS::update(float pitch, float altitude, float hgt_setpoint, float EAS_set
 	_control_param.tas_min = eas_to_tas * _equivalent_airspeed_min;
 	_control_param.pitch_max = pitch_limit_max;
 	_control_param.pitch_min = pitch_limit_min;
-	_control_param.throttle_trim = throttle_trim;
 	_control_param.throttle_max = throttle_setpoint_max;
 	_control_param.throttle_min = throttle_min;
 
+	if (throttle_trim > _control_param.throttle_trim / _stall_thr_trim_scale_to_prevent_stall) {
```

Comment:
@KonradRudin You could just set `_stall_thr_trim_scale_to_prevent_stall` in the if case and then get rid of the else and always compute `_control_param.throttle_trim = throttle_trim * _stall_thr_trim_scale_to_prevent_stall`